packages/codemods/.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules
 
 # Caches
 .cache
+.DS_Store
```

Comment:
Since this is a system-specific setting, it belongs to your `.gitignore_global` 🙂.

Reply:
removed